src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -19,7 +19,7 @@
 
     /** Default file path used if the user doesn't provide the file name. */
     public static final String DEFAULT_STORAGE_FILEPATH = "addressbook.txt";
-
+    public static final String DELETED_WARNING_MESSAGE ="WARNING, Storage File is deleted";
```

Comment:
As per the coding standard, there should be one space to the left and right of an equal sign.